libs-scala/gatling-utils/src/main/scala/com/daml/gatling/stats/Count.scala

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.gatling.stats
+
+// generic container for counted Gatling things
+// the T parameter is often Int, sometimes Double, Long for requestCount
+case class Count[T: Numeric](total: T, ok: T, ko: T) {
```

Comment:
```suggestion
final case class Count[T](total: T, ok: T, ko: T) {
```

The constraint should go on the `formatted` method instead.